sw/device/lib/dif/dif_adc_ctrl.c

```diff
@@ -513,3 +515,12 @@ dif_result_t dif_adc_ctrl_irq_cause_get_enabled(const dif_adc_ctrl_t *adc_ctrl,
 
   return kDifOk;
 }
+
+dif_result_t dif_adc_ctrl_wait_cdc_sync(const dif_adc_ctrl_t *adc_ctrl) {
+  if (adc_ctrl == NULL) {
+    return kDifBadArg;
+  }
+  // Wait 5 AON ticks.
+  busy_spin_micros(3 * 1000000 / (uint32_t)kClockFreqAonHz);
```

Comment:
This might be a stupid question, but I think this is "(3 * 1e6 / frequency_in_hz) microseconds". Won't that wait 3 cycles, rather than 5?

Reply:
Ah oops, this indeed should be a 5 not a 3. I am actually not entirely sure about which value to use, I based it off a comment in the ROM which says that it takes up to 4 AON cycles to synchronize.